### ISSUE TITLE
Can height be a negative value?

### ISSUE THREAD

**Issue by emres** [ErwinBaeyens/python-lander#6]
For example, looking at [Line 52](https://github.com/ErwinBaeyens/python-lander/blob/master/lander.py#L52), I wonder: Can height be a negative value?

I think, in this context, the answer is **no**. If that's the case, then it should be prevented. Of course, in the ideal case, this would be a compile-time error, but since we're not dealing with an advanced language, at least you can maybe use assertions.

See:

* http://stackoverflow.com/questions/944592/best-practice-for-python-assert
* http://stackoverflow.com/questions/5142418/what-is-the-use-of-assert-in-python
* https://wiki.python.org/moin/UsingAssertionsEffectively

**Comment by ErwinBaeyens** [comment]
yes it can but you'd be very dead.  I'll allow for it as the goal is to work tick based. And with a large enough tick value you can already be below the surface on the next tick. 
Unless I start modeling the compress-ability of the surface and the speed at the moment we hit the soil. 
For now it will be: at this point the height is  zero or less and if speed is more than x  yo are dead.  if the height is negative and you are considered to be dead it will be displayed as the depth of the crater you created for yourself  ;-)
